docs/marketplace/dev/feature-walkthroughs/202204.0/marketplace-product-offer-feature-walkthrough/marketplace-product-offer-feature-walkthrough.md

```diff
@@ -25,16 +25,17 @@ To learn more about the feature and to find out how end users use it, see [Marke
 
 The following table lists the main modules of the Marketplace Product Offer feature.
 
-| NAME | DESCRIPTION |
-| -------------------- | ---------- |
-| [MerchantProductOffer](https://github.com/spryker/merchant-product-offer) | Provides a collection of product offers by request. Extends `ProductOffer` with the merchant information. Used by `MerchantSwitcher` for enabling the merchant functionality.  |
-| [ProductOffer](https://github.com/spryker/product-offer) | Provides the main create-read-update product offer functionality.  |
-| [ProductOfferValidity](https://github.com/spryker/product-offer-validity) | Defines validity period for an offer. |
-| [Shop.MerchantProductOfferWidget](https://github.com/spryker-shop/merchant-product-offer-widget) | Provides merchant product offer information for the `spryker-shop`. |
+| NAME                                                                                                                | DESCRIPTION                                                                                                                                                                   |
+|---------------------------------------------------------------------------------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| [MerchantProductOffer](https://github.com/spryker/merchant-product-offer)                                           | Provides a collection of product offers by request. Extends `ProductOffer` with the merchant information. Used by `MerchantSwitcher` for enabling the merchant functionality. |
+| [ProductOffer](https://github.com/spryker/product-offer)                                                            | Provides the main create-read-update product offer functionality.                                                                                                             |
+| [ProductOfferValidity](https://github.com/spryker/product-offer-validity)                                           | Defines validity period for an offer.                                                                                                                                         |
+| [Shop.MerchantProductOfferWidget](https://github.com/spryker-shop/merchant-product-offer-widget)                    | Provides merchant product offer information for the `spryker-shop`.                                                                                                           |
+| [Shop.MerchantProductOfferWidgetExtension](https://github.com/spryker-shop/merchant-product-offer-widget-extension) | This module provides plugin interfaces to extend the `Shop.MerchantProductOfferWidget` module from the other modules.                                                                                                                                   |
```

Comment:
```suggestion
| [MerchantProductOfferWidget](https://github.com/spryker-shop/merchant-product-offer-widget)                    | Provides merchant product offer information for the `spryker-shop`.                                                                                                           |
| [MerchantProductOfferWidgetExtension](https://github.com/spryker-shop/merchant-product-offer-widget-extension) | This module provides plugin interfaces to extend the `MerchantProductOfferWidget` module from the other modules.                                                                                                                                   |
```
Lets be consistent with other feature walkthrough, they don't usually have `Shop` prefix